### PR TITLE
Release 0.1.0

### PR DESCRIPTION
**Pull request by ydrisR**
Integration of tidepool changes

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,17 @@
-FROM node:6.10.3-alpine
+FROM node:10.14.2-alpine
```
Comment (yvon-dblg) [inline Dockerfile]: Should it be 10.15.3 ?

File: Dockerfile [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+    apk --no-cache upgrade
 
 WORKDIR /app
 
-COPY . .
+COPY package.json package.json
 
-RUN yarn install
+RUN yarn install && \
+    yarn cache clean
```
Comment (yvon-dblg) [inline Dockerfile]: travis use npm and not yarn. Usually it is not recommended to use both program on the same project.

**Review by yvon-dblg — CHANGES_REQUESTED** [review]
Do we still need yarn.lock ? Since we use package-lock.json ?
See remarks on Dockerfile.

File: Dockerfile (hunk 1 above)
Comment (ydrisR) [reply Dockerfile]: why not ;)

**Review by yvon-dblg — APPROVED** [review]

**Pull request merged**